contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L2DepositedERC20.sol

```diff
@@ -21,139 +20,38 @@ import { OVM_CrossDomainEnabled } from "../../../libraries/bridge/OVM_CrossDomai
  * Compiler used: optimistic-solc
  * Runtime target: OVM
  */
-contract OVM_L2DepositedERC20 is iOVM_L2DepositedERC20, UniswapV2ERC20, OVM_CrossDomainEnabled {
-    
-    /*******************
-     * Contract Events *
-     *******************/
+contract OVM_L2DepositedERC20 is Abs_L2DepositedERC20, UniswapV2ERC20 {
```

Comment:
Maybe we should rename UniswapV2ERC20 to ERC20? We can recognize them in the comments on the file header, but it feels a bit odd to see the project explicitly called out here.

Reply:
Maybe PermitERC20?